src/components/pages/admin/PageProductImport/components/CSVFileImport.tsx

```diff
@@ -31,6 +31,9 @@ export default function CSVFileImport({ url, title }: CSVFileImportProps) {
       }
       const response = await axios({
         method: "GET",
+        headers: {
```

Comment:
I suggest using  [axios.interceptors](https://axios-http.com/docs/interceptors) to prevent nessesety to add authorization for every request manually